src/Talis/Persona/Client/ValidationErrors.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Talis\Persona\Client;
+
+abstract class ValidationErrors
```

Comment:
Validation Errors is a new class. Perhaps this would be better as ValidationResults, with a success included to stop the multiple return types above?